ceph_iscsi_config/lun.py

```diff
@@ -417,6 +428,10 @@ def allocate(self):
         so = self.lio_stg_object()
         if not so:
 
+            control = gen_control_string(self.controls)
```

Comment:
Nit: can all this logic just move down into ```add_dev_to_lio```? I ask because my gwcli snapshot handling PR utilizes ```add_dev_to_lio``` directly so it would be good if I don't need to dup this logic. This method should store the attributes in ```disk_attr``` and ```add_dev_to_lio``` could pull them out and merge w/ the defaults.